decidim-proposals/config/locales/en.yml

```diff
@@ -75,6 +82,11 @@ en:
           new:
             create: Create
             title: Create proposal
+        shared:
+          info_proposal:
+            body: Body
+            created_at: Date creation
+            proposal_votes_count: Nº votes
```

Comment:
Votes count

Reply:
Votes or notes?

Reply:
Votes. At least that's what the [issue](https://github.com/decidim/decidim/issues/2275) says

Reply:
Sorry, since the issue is about notes I wasn't sure if it was a typo! Could you please change the text to "Votes count" though?